contracts/modules/ProtocolSettings.sol

```diff
@@ -114,6 +116,15 @@ contract ProtocolSettings is State, ProtocolTokenUser, ProtocolSettingsEvents, M
 		emit SetLockedSOVAddress(msg.sender, oldLockedSOVAddress, newLockedSOVAddress);
 	}
 
+	function setTradingRebateRewardsBasisPoint(uint256 newBasisPoint) external onlyOwner whenNotPaused {
+		require(newBasisPoint <= 10000, "value too high");
```

Comment:
lockedSOV only allows 9999, does it not? So, we should have the same restriction here.